templates/workspaces/base/terraform/network/security.tf

```diff
@@ -172,6 +172,23 @@ resource "azurerm_network_security_rule" "allow-inbound-from-resourceprocessor"
   source_port_range = "*"
 }
 
+
+resource "azurerm_network_security_rule" "allow-inbound-from-airlockprocessor" {
```

Comment:
to where? Is services correct?

Reply:
I copied paster from the resource processor rule 5  lines above. as it should behave the same.
yes, services is correct.